website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/liststeps-endpoint-deprecation.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "List Steps API endpoint deprecation warning"
+id: "liststeps-endpoint-deprecation.md"
+description: "List Steps API deprecation"
+sidebar_label: "Deprecation: List Steps API endpoint"
+tags: [Sept-15-2022]
+---
+
+On October 14th, 2022 we are deprecating the [List Steps](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs/operation/listSteps) API endpoint. From October 14th, any GET requests to this endpoint will fail. Please prepare to move off of the List Steps endpoint as soon as possible. 
```

Comment:
```suggestion
On October 14th, 2022 dbt Labs is deprecating the [List Steps](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs/operation/listSteps) API endpoint. From October 14th, any GET requests to this endpoint will fail. Please prepare to stop using the List Steps endpoint as soon as possible. 
```

Reply:
sugg:
- "we" can be ambiguous. is it "dbt labs" that is being referred or is it "dbt labs + the reader" that's being referred? this ambiguity can make it harder to translate. don't avoid using it (as it makes sense to use it in certain situations) but use it sparingly. 
- "move off" might be a USAn colloquialism, not totally sure, but i do wonder how that'll translate to international users

Reply:
ah yes great flag Ly, thank you for that and have committed your suggestions!